cli/src/wallet_cli/mod.rs

```diff
@@ -308,7 +307,7 @@ impl FromStr for OutputSelector {
 
 // `accounts` command
 pub async fn accounts_command(wallet: &Wallet) -> Result<(), Error> {
-    print_outputs(wallet.accounts().await, "Accounts:").await
+    print_outputs(wallet.data().await.accounts().cloned().collect(), "Accounts:")
```

Comment:
we really need to clone here?

Reply:
oef .cloned() all over, not nice :/

Reply:
The clones were implicitly done by the non-iterative methods so yeah we need to explicitly clone now. But it means that it's way more efficient in the majority of the cases.